templates/woocommerce-dependency-notice.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Admin View: Dependency notice
+ *
+ * @since 1.1.9
+ *
+ * @package WePOS
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+    exit;
+}
+
+/**
+ * @var bool $has_woocommerce
+ * @var bool $woocommerce_installed
+ */
+?>
+<?php if ( ! $has_woocommerce ): ?>
```

Comment:
Don't check the logic in here, check where it was called. If doesn't have WC, include this file.